src/net/l4/tcp.c

```diff
@@ -54,12 +56,6 @@ EMBOX_NET_PROTO(ETH_P_IPV6, IPPROTO_TCP, tcp_rcv,
 #define MODOPS_VERIFY_CHKSUM OPTION_GET(BOOLEAN, verify_chksum)
 
 #define TCP_DEBUG 0
```

Comment:
Yes,
`packet_print` depends of `debug_print`, and both depend of `TCP_DEBUG` macro.
I just changed `printf` function by `log_debug` in `debug_print` function, but I can reduce `TCP_DEBUG` by changing all `packet_print` and `debug_print` directly by `log_debug`.
Schould I do it?